spec/jekyll-readme-index/generator_spec.rb

```diff
@@ -310,6 +310,46 @@
         end
       end
     end
+
+    context "with a readme and an override" do
```

Comment:
This is honestly a pretty half-asses attempt at adding the tests. This PR is sort of a testing-the-waters type of a thing to see if this would even be considered for inclusion so I didn't pour a whole lotta effort to making the tests as nice and complete as the rest of the spec file. I will amend if I get an indication this would be considered to inclusion! 😇